arch/RISCV/riscv_ast.gen.inc

```diff
@@ -0,0 +1,2712 @@
+#include <stdint.h>
+
+#include <stddef.h>
+
+#include <string.h>
+
```

Comment:
maybe add at the beginning a multiline comment saying that this SAIL generated from the riscv repository YYYYYY with commit UUUUU

Reply:
Also the copyright. Will be necessary for the SPDX in the future: https://github.com/capstone-engine/capstone/issues/2132